relay-metrics/src/statsd.rs

```diff
@@ -174,5 +174,12 @@ pub fn metric_name_tag(value: &str) -> &str {
         return "d:transactions/measurements.*";
     }
 
+    if value.starts_with("s:spans/") {
+        return "s:spans/";
+    }
+    if value.starts_with("d:spans/") {
+        return "d:spans/";
+    }
```

Comment:
We have several distribution metrics. I decided to not add the full MRIs to the list above to control the cost further, but I'll revisit this decision if we need more details.